tests/browser/donations/create_contact_donation.robot

```diff
@@ -7,19 +7,16 @@ Suite Teardown  Delete Records and Close Browser
 *** Test Cases ***
 
 Create Donation from a Contact
-    #1 contact HouseHold Validation
-    ${contact_id} =  Create Contact with Email
-    &{contact} =  Salesforce Get  Contact  ${contact_id}
-    Header Field Value    Account Name    &{contact}[LastName] Household
-    Scroll Page To Location    0    500
-    Sleep    5
+    &{contact} =  API Create Contact    Email=skristem@robot.com
+    Go To Record Home  &{contact}[Id]
+    Page Scroll to Locator    record.related.title    Volunteer Hours
     Click Special Related List Button   Opportunities    New Contact Donation
     Choose Frame    New Opportunity
     Click Element    p3
     Select Option    Donation    
-    Click Element    //input[@title='Continue']
-    Sleep    5  
+    Click Button With Value    Continue
```

Comment:
I think just "Click Button" will work. `value` is one of the default locator types for this keyword.